src/search.h

```diff
@@ -49,6 +49,12 @@ struct Stack {
   Value staticEval;
   int statScore;
   int moveCount;
+
+  Stack(): pv(NULL), continuationHistory(NULL), currentMove(MOVE_NONE),
```

Comment:
I know I will be called a nitpicking code style zealot again, but you might want to use `nullptr` instead of `NULL` here :)

But actually... you can leave out all the nullptr/zero/MOVE_NONE intialization, since all the fields should be already default-constructed (for primitive types this means to a 0 value) by default once you explictly define the constructor. So it should be enough to just write:
```
Stack() { }
```

Reply:
Or wait... maybe I'm wrong, because valgrind reports some uses of uninitialized values in your patch, presumably because you forgot to initialize `staticEval` in the constructor...